src/pylhe/__init__.py

```diff
@@ -168,8 +171,8 @@ def readLHE(thefile):
                 for p in particles:
                     particle_objs += [LHEParticle.fromstring(p)]
                 yield LHEEvent(eventinfo, particle_objs)
-    except ET.ParseError:
-        print("WARNING. Parse Error.")
+    except ET.ParseError as e:
+        print("WARNING. Parse Error:", e)
```

Comment:
We should have an extra space between the `:` and the `"` so you don't have text smashed up against each other. 
```suggestion
    except ET.ParseError as e:
        print("WARNING. Parse Error: ", e)
```

Reply:
The comma inserts a blank.